### PR TITLE
Fix Visual bug on rundown

### LINKED ISSUE #676: Visual Bug: Rundown editor
image

When trying to edit an event that is in the "past" in the current rundown you get this visual bug and are not able to access the options that are behind the other element.

The opacity change for the "past" rundown entry is most likely to cause this issue.

### PR DESCRIPTION
**Pull request by kellhogs**
Im almost certain there is a smarter way to solve this ..... but this works too

closes #676

### CODE REVIEW

File: apps/client/src/features/rundown/event-block/EventBlock.module.scss [hunk 1]
```diff
@@ -54,7 +54,16 @@ $skip-opacity: 0.1;
   }
 
   &.past:not(.skip) {
```
Comment (cpvalente) [inline apps/client/src/features/rundown/event-block/EventBlock.module.scss:56]: nice!

if what we are trying to do is stop the opacity in an element, could we make it explicit that way?
```
&.past:not(.skip) {
  opacity: 0.4;

  .elementWithoutOpacity {
    opacity: 1;
  }
}
```

Comment (kellhogs) [reply apps/client/src/features/rundown/event-block/EventBlock.module.scss:56]: for some weird reason this doesn't work. The element i'm sparing with the opacity is the .eventActions

if i write it like you proposed, it doesn't overwrite the opacity on these elements. Even with !Important

someone who is smarter with css than me could probably figure this out.

Comment (cpvalente) [reply apps/client/src/features/rundown/event-block/EventBlock.module.scss:56]: hmm, probably, but probably not worth losing too much sleep over it
Would you mind adding a comment as to why this is necessary like?
/* we stop the eventActions from having opacity to fix issue with dropdown drawing order */

Comment (kellhogs) [reply apps/client/src/features/rundown/event-block/EventBlock.module.scss:57]: all done

**Review by cpvalente — APPROVED** [review]
🚀

**Pull request merged**